src/dfx-core/src/config/model/network_descriptor.rs

```diff
@@ -1,9 +1,12 @@
+use slog::Logger;
+use url::Url;
+
 use crate::config::model::dfinity::NetworkType;
 use crate::config::model::dfinity::{DEFAULT_IC_GATEWAY, DEFAULT_IC_GATEWAY_TRAILING_SLASH};
 use crate::config::model::local_server_descriptor::LocalServerDescriptor;
 use crate::error::network_config::NetworkConfigError;
 use crate::error::network_config::NetworkConfigError::{NetworkHasNoProviders, NetworkMustBeLocal};
-
+use crate::error::uri::UriError;
 use std::path::{Path, PathBuf};
```

Comment:
We seem to have a convention of putting `use` statements for the current repo first, then a blank line, then `use` statements for external crates.
```suggestion

use std::path::{Path, PathBuf};
use slog::Logger;
use url::Url;
```